crypto/store/store_local.h

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 1995-2016 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the OpenSSL license (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#include "internal/thread_once.h"
+#include <openssl/dsa.h>
+#include <openssl/engine.h>
+#include <openssl/evp.h>
+#include <openssl/lhash.h>
+#include <openssl/x509.h>
+#include <openssl/store.h>
+
+/******************************************************************************
+ *
+ * STORE_INFO stuff
+ *
+ *****/
```

Comment:
Well, you've got two strong views against, and your strong view in favor.  Take it to the team I guess.

None of our other "block headers" work this way, do they?